### PR TITLE
Create ui-c8y-1020-0-0-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre

Ticket: [MTM-58410](https://cumulocity.atlassian.net/browse/MTM-58410)
Original PR: [6288](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6288)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Data points graph realtime disabled when tab is in background (#6288)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Disable real-time updates for data point graphs when browser tab is in background
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58410
+version: 1020.0.0
+---
+Data points graph realtime disabled when tab is in background (#6288)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve performance and reduce unnecessary resource consumption, real-time updates for data point graphs are now disabled when the browser tab containing the Cumulocity application is in the background or not visible to the user. This change optimizes system resources by avoiding the processing of real-time updates for graphs that are not currently being viewed. Users will notice that data point graphs will not update in real-time when the application is not in focus, but will resume updating once the tab is brought back to the foreground.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@jdreSAG I copied the approved text from the related 2024 release note.

**Review by jdreSAG — APPROVED** [review]

**Pull request merged**